src/qtiCommonRenderer/renderers/interactions/ExtendedTextInteraction.js

```diff
@@ -703,13 +703,12 @@ function inputLimiter(interaction) {
             };
 
             const handleCompositionEnd = e => {
-                e.preventDefault();
```

Comment:
@KirylHatalski Was there a reason why `e.preventDefault()` & `blur` were added? It seems they have no impact, good or bad.